Engine/Shared/Ecs/Src/Registry.hpp

```diff
@@ -188,10 +188,10 @@ class Registry {
          */
         void setComponentsArrays(std::unordered_map<std::string, std::any> componentsArrays, std::unordered_map<std::string, remove_func_t> removeFunctions);
 
+        std::unordered_map<std::string, std::any>                           _components_arrays;         // Array of components.
```

Comment:
Why making it public when there's a getter for it ?

Reply:
Oups sorry it was for debug purpose.